src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -642,28 +646,25 @@ public static bool Equals(object first, object second, bool ignoreCase, IFormatP
 
             if (first == null)
             {
-                return IsNullLike(second);
+                if (second == null) return true;
+                return false;
```

Comment:
`return second == null;`

Reply:
There are other nice syntax/formatting changes made in the original PRs that I wish we could keep ... but that involves too much manual changes that I afraid would introduce mistakes.
This one is simple (and annoying 😄) enough. Will update.

Reply:
@daxian-dbw if y'all want I can always do a cleanup-only PR here, I was wanting to clean up quite a bit more here anyway 😉

Reply:
That would be awesooooooome! 😃 Thank you, @vexx32

Reply:
Only some small PRs :-)

Reply:
@iSazonov how about one PR per file, and if it gets too big I'll categorise changes per commit 😁

Reply:
Sounds like a plan 👍